.github/workflows/python-package-conda.yml

```diff
@@ -59,6 +63,7 @@ jobs:
           source activate genomics.py3
           python -m pytest --ignore test/unit/utils/test_db_access.py --ignore test/unit/single_cell_qc test/unit
           python -m pytest test/system
+          python -m pytest ugbio_utils/src/*/tests
```

Comment:
I'm not sure we want to run the submodule tests on each push to variant calling (or even run submodule tests at all here)

Reply:
that's the spirit :) 
We want to be sure that ugbio modules work in the genomics.py3 environment (where we do research)